From 7f2c5bfa8f90e6b4dd04102979695576b3881164 Mon Sep 17 00:00:00 2001
From: Michael Fladischer <fladi@debian.org>
Date: Thu, 8 Oct 2015 08:50:28 -0700
Subject: Remove remote hosted logo image from documentation

 This is a potential breach of privacy for the user viewing the documentation
 from his local filesystem.
Forwarded: not-needed
Last-Update: 2015-07-07
Patch-Name: privacy.patch
---
 docs/.templates/sidebarlogo.html | 3 ---
 docs/conf.py                     | 2 +-
 docs/index.rst                   | 2 --
 docs/introduction.rst            | 2 --
 4 files changed, 1 insertion(+), 8 deletions(-)
 delete mode 100644 docs/.templates/sidebarlogo.html

diff --git a/docs/.templates/sidebarlogo.html b/docs/.templates/sidebarlogo.html
deleted file mode 100644
index 1dd18ab..0000000
--- a/docs/.templates/sidebarlogo.html
+++ /dev/null
@@ -1,3 +0,0 @@
-<p class="logo"><a href="{{ pathto(master_doc) }}">
-  <img class="logo" src="http://cloud.github.com/downloads/ask/celery/celery_favicon_128.png" alt="Logo"/>
-</a></p>
diff --git a/docs/conf.py b/docs/conf.py
index 7d12486..c17d4e2 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -76,7 +76,7 @@ html_theme = 'celery'
 html_theme_path = ['_theme']
 html_sidebars = {
     'index': ['sidebarintro.html', 'sourcelink.html', 'searchbox.html'],
-    '**': ['sidebarlogo.html', 'localtoc.html', 'relations.html',
+    '**': ['localtoc.html', 'relations.html',
            'sourcelink.html', 'searchbox.html'],
 }
 
diff --git a/docs/index.rst b/docs/index.rst
index 4a73a4e..0ecbb81 100644
--- a/docs/index.rst
+++ b/docs/index.rst
@@ -1,5 +1,3 @@
-.. image:: http://cloud.github.com/downloads/ask/celery/celery_128.png
-
 ===============================
  Celery Integration for Django
 ===============================
diff --git a/docs/introduction.rst b/docs/introduction.rst
index f2ab2ea..533e1dd 100644
--- a/docs/introduction.rst
+++ b/docs/introduction.rst
@@ -2,8 +2,6 @@
  django-celery - Celery Integration for Django
 ===============================================
 
-.. image:: http://cloud.github.com/downloads/celery/celery/celery_128.png
-
 :Version: 3.1.9
 :Web: http://celeryproject.org/
 :Download: http://pypi.python.org/pypi/django-celery/
